### ISSUE TITLE
LineageOS 19.1/android 12 not working on sdcard

### ISSUE THREAD

**Issue by nutpantz** [Xposed-Modules-Repo/com.github.dan.nostoragerestrict#19]
i cannot see /storage/(sdcardname)/android/data ( the SD CARD)
LineageOS 19.1 samsung s7

**Comment by DanGLVK** [comment]
> i cannot see /storage/(sdcardname)/android/data ( the SD CARD) LineageOS 19.1 samsung s7

What file manager exactly? Did it ask for permission to access that folder? My modules merely allows granting access to those folders, it won't automatically allow any app to access them (if it doesn't ask for permission)

**Comment by nutpantz** [comment]
I was using total commander 2.50
It did not ask for folder permission.
I have root and total commander has root permission. It usually asks for folder permission.

**Comment by DanGLVK** [comment]
> I was using total commander 2.50 It did not ask for folder permission. I have root and total commander has root permission. It usually asks for folder permission.
> […](#)
> On July 21, 2024 12:22:06 PM PDT, DanGLES3 ***@***.***> wrote: > i cannot see /storage/(sdcardname)/android/data ( the SD CARD) LineageOS 19.1 samsung s7 What file manager exactly? Did it ask for permission to access that folder? -- Reply to this email directly or view it on GitHub: [#19 (comment)](https://github.com/Xposed-Modules-Repo/com.github.dan.nostoragerestrict/issues/19#issuecomment-2241747904) You are receiving this because you authored the thread. Message ID: ***@***.***>

If you have root permission you can access /android/data by going to /data/media/0 instead

**Comment by nutpantz** [comment]
I'm looking to access the external sdcard

I don't not appear to have /storage/media
 Just "/storage/0/" "/storage/sdcard name/” and a link that goes to "/storage/self/primary/” which is the same as 0

At least "media" is not showing up in total commander
There is a /mnt/media_rw/ that works for the sd card

But i was hoping for the default "/storage/sdcard name/” path.

**Comment by DanGLVK** [comment]
> I'm looking to access the external sdcard I don't not appear to have /storage/media Just "/storage/0/" "/storage/sdcard name/” and a link that goes to "/storage/self/primary/” which is the same as 0 At least "media" is not showing up in total commander There is a /mnt/media_rw/ that works for the sd card But i was hoping for the default "/storage/sdcard name/” path.
> […](#)
> On July 21, 2024 4:37:41 PM PDT, DanGLES3 ***@***.***> wrote: > I was using total commander 2.50 It did not ask for folder permission. I have root and total commander has root permission. It usually asks for folder permission. > […](#) > On July 21, 2024 12:22:06 PM PDT, DanGLES3 ***@***.***> wrote: > i cannot see /storage/(sdcardname)/android/data ( the SD CARD) LineageOS 19.1 samsung s7 What file manager exactly? Did it ask for permission to access that folder? -- Reply to this email directly or view it on GitHub: [#19 (comment)]([#19 (comment)](https://github.com/Xposed-Modules-Repo/com.github.dan.nostoragerestrict/issues/19#issuecomment-2241747904)) You are receiving this because you authored the thread. Message ID: ***@***.***> If you have root permission you can access /android/data by going to /data/media/0 instead -- Reply to this email directly or view it on GitHub: [#19 (comment)](https://github.com/Xposed-Modules-Repo/com.github.dan.nostoragerestrict/issues/19#issuecomment-2241815680) You are receiving this because you authored the thread. Message ID: ***@***.***>

Uhh, my module doesn't have anything to do with accessing the external sdcard tho

**Comment by nutpantz** [comment]
I installed root explorer and it seems to work just fine so it must be an issue with total commander.

**Comment by nutpantz** [comment]
Well that's strange then, because it's letting me choose the sd card Android/"data" folder when i could not without it.

**Comment by DanGLVK** [comment]
> Well that's strange then, because it's letting me choose the sd card Android/"data" folder when i could not without it.
> […](#)
> On July 21, 2024 4:59:42 PM PDT, DanGLES3 ***@***.***> wrote: > I'm looking to access the external sdcard I don't not appear to have /storage/media Just "/storage/0/" "/storage/sdcard name/” and a link that goes to "/storage/self/primary/” which is the same as 0 At least "media" is not showing up in total commander There is a /mnt/media_rw/ that works for the sd card But i was hoping for the default "/storage/sdcard name/” path. > […](#) > On July 21, 2024 4:37:41 PM PDT, DanGLES3 ***@***.***> wrote: > I was using total commander 2.50 It did not ask for folder permission. I have root and total commander has root permission. It usually asks for folder permission. > […](#) > On July 21, 2024 12:22:06 PM PDT, DanGLES3 ***@***.***> wrote: > i cannot see /storage/(sdcardname)/android/data ( the SD CARD) LineageOS 19.1 samsung s7 What file manager exactly? Did it ask for permission to access that folder? -- Reply to this email directly or view it on GitHub: [#19 (comment)]([#19 (comment)]([#19 (comment)](https://github.com/Xposed-Modules-Repo/com.github.dan.nostoragerestrict/issues/19#issuecomment-2241747904))) You are receiving this because you authored the thread. Message ID: ***@***.***> If you have root permission you can access /android/data by going to /data/media/0 instead -- Reply to this email directly or view it on GitHub: [#19 (comment)]([#19 (comment)](https://github.com/Xposed-Modules-Repo/com.github.dan.nostoragerestrict/issues/19#issuecomment-2241815680)) You are receiving this because you authored the thread. Message ID: ***@***.***> Uhh, my module doesn't have anything to do with accessing the sdcard tho -- Reply to this email directly or view it on GitHub: [#19 (comment)](https://github.com/Xposed-Modules-Repo/com.github.dan.nostoragerestrict/issues/19#issuecomment-2241822622) You are receiving this because you authored the thread. Message ID: ***@***.***>

Let me rephrase it, it can only allow granting access to the sdcard if the app goes through the storage access framework to do so, otherwise it does nothing for apps which don't use it
